shared/challenge_builder.py

```diff
@@ -56,6 +18,9 @@ def __init__(self, geometry, properties={}):
     @classmethod
     def withId(cls, osmType, osmId, geometry, properties):
         properties["@id"] = f"{osmType}/{osmId}"
+        # Fix polygons beeing not a 3-dimensional array
+        if geometry["type"] == "Polygon" and len(geometry["coordinates"]) != 1:
+            geometry["coordinates"] = [geometry["coordinates"]]
```

Comment:
This shouldn't be part of this class. Please move this to another place (e.g. queryElementsAsGeoJSON)

Reply:
Moved it to init of the same class. Everywhere else, it has no effect.